package/boot/rkbin/Makefile

```diff
@@ -0,0 +1,47 @@
+# SPDX-License-Identifier: GPL-2.0-only
+#
+# Copyright (C) 2021-2023 ImmortalWrt.org
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=rkbin
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL=https://github.com/rockchip-linux/rkbin.git
```

Comment:
I found this pull request to add rk3568 support to upstream tf-a:
https://review.trustedfirmware.org/c/TF-A/trusted-firmware-a/+/16952

It is probably better to go with this binary blob for now.

Reply:
> It is probably better to go with this binary blob for now.

Yes. My friend had tested it a few months ago and found the upstream one still had stabilize issues.